src/main/java/ch/wisv/areafiftylan/products/service/OrderServiceImpl.java

```diff
@@ -187,10 +213,14 @@ public Order updateOrderStatus(Long orderId) {
 
     @Override
     public void adminApproveOrder(Long orderId) {
```

Comment:
Let's leave this for now, and make an issue to replace with the functionality you suggest